app/views/contacts/_organisation_contact.html.erb

```diff
@@ -18,21 +18,21 @@
     <% if contact.email.present? || contact.contact_form_url.present? || contact.contact_numbers.any? %>
       <div class="govuk-grid-column-one-third">
         <% if contact.email.present? %>
-          <p class="govuk-!-margin-bottom-4">
+          <p class="govuk-body govuk-!-margin-bottom-4">
```

Comment:
We need the govuk-body class applied to the address tag on line 13 as well, currently the font is wrong:https://www.integration.publishing.service.gov.uk/world/organisations/british-antarctic-territory